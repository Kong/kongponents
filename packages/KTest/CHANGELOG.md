# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

# 4.0.0 (2021-09-03)


### Bug Fixes

* **automation:** add publish token ([#274](https://github.com/Kong/kongponents/issues/274)) ([108899d](https://github.com/Kong/kongponents/commit/108899d17b4c428154133c98400728e9cdcfa4f1))
* **automation:** remove 2FA ([#275](https://github.com/Kong/kongponents/issues/275)) ([ce9ca83](https://github.com/Kong/kongponents/commit/ce9ca83cf763eeeb1ad98535127807e75aa1e109))
* **automation:** test automation token ([#273](https://github.com/Kong/kongponents/issues/273)) ([0310548](https://github.com/Kong/kongponents/commit/0310548e89dace0b484236617fe96c79999da99d))


* feat(KTest) change component, test build ([945c06f](https://github.com/Kong/kongponents/commit/945c06fdb2972ea884e6f52b20937c4dba45ba72))


### BREAKING CHANGES

* change test component





## [1.0.7](https://github.com/Kong/kongponents/compare/@kongponents/ktest@1.0.6...@kongponents/ktest@1.0.7) (2020-12-01)


### Bug Fixes

* **automation:** remove 2FA ([#275](https://github.com/Kong/kongponents/issues/275)) ([ce9ca83](https://github.com/Kong/kongponents/commit/ce9ca83cf763eeeb1ad98535127807e75aa1e109))





## [1.0.6](https://github.com/Kong/kongponents/compare/@kongponents/ktest@1.0.5...@kongponents/ktest@1.0.6) (2020-12-01)


### Bug Fixes

* **automation:** add publish token ([#274](https://github.com/Kong/kongponents/issues/274)) ([108899d](https://github.com/Kong/kongponents/commit/108899d17b4c428154133c98400728e9cdcfa4f1))





## [1.0.5](https://github.com/Kong/kongponents/compare/@kongponents/ktest@1.0.4...@kongponents/ktest@1.0.5) (2020-12-01)


### Bug Fixes

* **automation:** test automation token ([#273](https://github.com/Kong/kongponents/issues/273)) ([0310548](https://github.com/Kong/kongponents/commit/0310548e89dace0b484236617fe96c79999da99d))





## [1.0.4](https://github.com/Kong/kongponents/compare/@kongponents/ktest@1.0.3...@kongponents/ktest@1.0.4) (2020-03-06)

**Note:** Version bump only for package @kongponents/ktest





## [1.0.3](https://github.com/Kong/kongponents/compare/@kongponents/ktest@1.0.1...@kongponents/ktest@1.0.3) (2020-01-16)

**Note:** Version bump only for package @kongponents/ktest





## [1.0.1](https://github.com/Kong/kongponents/compare/@kongponents/ktest@1.0.0...@kongponents/ktest@1.0.1) (2019-11-12)

**Note:** Version bump only for package @kongponents/ktest





# [1.0.0](https://github.com/Kong/kongponents/compare/@kongponents/ktest@0.2.2...@kongponents/ktest@1.0.0) (2019-11-08)


* feat(KTest) change component, test build ([945c06f](https://github.com/Kong/kongponents/commit/945c06fdb2972ea884e6f52b20937c4dba45ba72))


### BREAKING CHANGES

* change test component





## [0.2.2](https://github.com/Kong/kongponents/compare/@kongponents/ktest@0.2.1...@kongponents/ktest@0.2.2) (2019-11-08)

**Note:** Version bump only for package @kongponents/ktest





## 0.2.1 (2019-11-06)

**Note:** Version bump only for package @kongponents/ktest





# [0.2.0](https://github.com/Kong/kongponents/compare/@kongponents/ktest@0.1.5...@kongponents/ktest@0.2.0) (2019-11-06)


### Bug Fixes

* **ktest:** minor bump ([4758ae5](https://github.com/Kong/kongponents/commit/4758ae5))


### Features

* **ktest:** major bump ([0ed98a4](https://github.com/Kong/kongponents/commit/0ed98a4))





## 0.1.5 (2019-11-06)

**Note:** Version bump only for package @kongponents/ktest
