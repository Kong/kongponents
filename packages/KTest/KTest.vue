<template>
  <div :style="{color, fontFamily, fontSize}"><slot/></div>
</template>

<script>
export default {
  name: 'KTest',
  props: {
    fontFamily: {
      type: String,
      required: false,
      default: 'monospace'
    },
    color: {
      type: String,
      required: false,
      default: '#0089eb'
    },
    fontSize: {
      type: String,
      required: false,
      default: '2rem'
    }
  }
}
</script>
