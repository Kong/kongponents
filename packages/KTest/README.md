# KTest

⚠️ This package exists to test out our CI process (publishing, tagging, lerna,
etc.). It's useless otherwise.
