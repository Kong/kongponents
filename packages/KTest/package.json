{
  "name": "@kongponents/ktest",
  "version": "5.0.0",
  "description": "Testing 1, 2, 3, 4...",
  "main": "dist/KTest.umd.min.js",
  "componentName": "KTest",
  "source": "KTest.vue",
  "files": [
    "dist",
    "KTest.vue"
  ],
  "repository": {
    "type": "git",
    "url": "git+https://github.com/Kong/kongponents.git"
  },
  "author": "Kong Inc",
  "license": "Apache-2.0",
  "bugs": {
    "url": "https://github.com/Kong/kongponents/issues"
  },
  "homepage": "https://github.com/Kong/kongponents#readme",
  "publishConfig": {
    "access": "public"
  }
}
