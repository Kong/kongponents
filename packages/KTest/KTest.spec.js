import { mount } from '@vue/test-utils'
import KTest from '@/KTest/KTest'

describe('KTest', () => {
  it('matches snapshot', () => {
    const wrapper = mount(KTest)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
