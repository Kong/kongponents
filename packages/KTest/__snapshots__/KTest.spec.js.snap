// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KTest matches snapshot 1`] = `<div style="color: rgb(0, 137, 235); font-family: monospace; font-size: 2rem;"></div>`;
